search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -55,25 +55,25 @@ function setCountryTrends(countryCode) {
  * @param {string} countryCode Two letter country code for selected country.
  */
 function displayTopResultsForCurrentTrend(countryCode) {
-  let topic = getCurrentTrend;
-  let topicData = getCurrentCustomSearchData();
-  let date = new Date(topicData.timestamp);
-  let resultElement = document.getElementById('search-results-tab');
+  let dataByCountry = getCurrentCustomSearchData().dataByCountry;
+  let date = new Date(getCurrentCustomSearchData().timestamp);
+  let resultElement =  document.getElementById('search-results-tab');
```

Comment:
Delete extra space.

Reply:
Did it but not sure if that's what you mean